.env.local.example

```diff
@@ -0,0 +1,9 @@
+# Algolia
+NEXT_PUBLIC_ALGOLIA_APP_ID=
+NEXT_PUBLIC_ALGOLIA_SEARCH_API_KEY=
+NEXT_PUBLIC_ALGOLIA_BASE_SEARCH_INDEX_NAME=
+
+# GitHub API
+# Note: this token expires on Dec 16, 2023
```

Comment:
I think I would leave this line out. This references one token, but this could change without this file being updated, and also wouldn't apply is someone is running a local build.

```suggestion
```